microsoft-edge/devtools-guide-chromium/console/console-dom-interaction.md

```diff
@@ -5,290 +5,345 @@ author: MSEdgeTeam
 
-1. Press **Ctrl+Shift+J** (Windows, Linux) or **Command+Option+J** (macOS).  The Console opens in DevTools, next to the present webpage.
+1. Go to the webpage that you want to change. For example, open the [DevTools Console tool DOM interactions demo page](https://microsoftedge.github.io/Demos/devtools-console-dom-interactions/) in a new tab or window.
+
+1. Press **Ctrl+Shift+J** (Windows, Linux) or **Command+Option+J** (macOS).  The **Console** tool opens in DevTools, next to the webpage.
 
 1. Paste the following code into the **Console**:
 
    ```javascript
-   document.querySelector('header').style.border = '2em solid green'
+   document.querySelector("h1").style.backgroundColor = "lightgrey";
    ```
 
-   A green border appears around the header:
+   A grey background appears behind the page title:
 
-   ![Adding a border to an element by using the Console](./console-dom-interaction-images/console-dom-add-border.png)
+   ![The Console tool, showing the above JavaScript code, and the webpage showing the grey title](./console-dom-interaction-images/set-style.png)
 
 
-### Get a direct reference an element
+#### Get a reference to an element
 
-Depending on the complexity of the webpage, it can be daunting to find the right element to manipulate.  But you can use the **Inspect** tool to help you.  Suppose you want to manipulate the **Documentation** region within the header of the rendered page:
+Depending on the complexity of the webpage, it can be daunting to find the right element to change.  But you can use the **Inspect** tool to help you.  Suppose you want to change the text of the first link within the **Section 1** element of the rendered page:
 
-![Display the element that you inspect on the screen](./console-dom-interaction-images/console-dom-highlight-documentation.png)
+![The link that you want to change in the page](./console-dom-interaction-images/link-to-change.png)
 
+To get a reference to the link element that you want to change:
 
-To get a direct reference to the element that you want to manipulate:
+1. Go to the webpage that you want to inspect. For example, open the [DevTools Console tool DOM interactions demo page](https://microsoftedge.github.io/Demos/devtools-console-dom-interactions/) in a new tab or window.
 
-1. In DevTools, click the **Inspect** tool, and then in the rendered webpage, hover over the element:
+1. Press **Ctrl+Shift+J** (Windows, Linux) or **Command+Option+J** (macOS).  The **Console** tool opens in DevTools, next to the webpage.
 
-   ![To select an element, use the Inspect tool](./console-dom-interaction-images/console-dom-use-inspector-to-get-element.png)
+1. Click the **Inspect** tool, and then in the rendered webpage, hover over the link element:
```

Comment:
```suggestion
1. Click the **Inspect** (![Inspect tool icon](./console-dom-interaction-images/inspect-tool-icon.png)) tool, and then in the rendered webpage, hover over the link element:
```
mid-sentence png would be nice here, since no text label on this special tool icon.